ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/map.view.js

```diff
@@ -459,7 +459,12 @@ const View = Marionette.LayoutView.extend({
       .get('preferences')
       .get('resultFilter')
     if (resultFilter) {
-      this.handleFilter(CQLUtils.transformCQLToFilter(resultFilter), '#c89600')
+      this.handleFilter(
+        CQLUtils.transformCQLToFilter(
```

Comment:
this is weird; we convert it and convert it again?

Reply:
see these comments https://github.com/codice/ddf/pull/5436#discussion_r332641217

Reply:
Might be good to add a comment summarizing the previous discussion